test/remindat.coffee

```diff
@@ -0,0 +1,19 @@
+chai = require 'chai'
+assert = chai.assert
+
+Helper = require 'hubot-test-helper'
+helper = new Helper('../scripts')
+
+describe 'Reminder formats', ->
+  room = null
+  beforeEach ->
+    room = helper.createRoom()
+    room.lastMessageText = ->
+      [...,last] = room.messages
+      last[1]
+
```

Comment:
I'd prefer not to define new method for testing to object.

how about is ? 

```
assert.deepEqual ["hubot", "I'll remind you to do task at Thu Aug 17 2513 12:00:00 GMT+0000 (UTC)"], room.messages[1]
```

it's simple.